### ISSUE TITLE
Missing volumes param in docker-compose.yml

### ISSUE THREAD

**Issue by rafaelmaia8384** [arnidan/nsfw-api#43]
The correct docker-compose.yml should be the following:

```
version: "3.9"

services:
  nsfw-api:
    image: "ghcr.io/arnidan/nsfw-api:latest"
    ports:
      - "3000:3000"
    restart: always
    volumes:
      - ./model:/model
```

**Comment by arnidan** [comment]
Hello!

Why do you need to mount the model folder and not use bundled one? 
It can be required for your needs but not in common.

**Comment by arnidan** [comment]
Hello again

Please, check if the fix of #48 works for you
